### DISCUSSION TITLE
WebRTC Integration

### DISCUSSION THREAD

**Discussion by Artekus** [blakeblackshear/frigate#1542] [Ideas]
Would you consider adding the option for WebRTC in addition the existing RTMP? 
Would you consider also adding WebRTC as a camera source? Given ffmpeg is utilized, I'm not sure this is possible but thought I'd ask.

Thank you for all the work you all put into this!

**Comment by blakeblackshear** [comment]
When I have looked into this in the past, it seems to be very complicated. The upcoming version uses jsmpeg for near realtime viewing and it's much more efficient than mjpeg. Not as good as webrtc, but probably good enough.
